Makefile

```diff
@@ -78,7 +78,8 @@ foam3_MAVEN_DEPS = \
 	com.google.appengine:appengine-api-1.0-sdk:1.9.24 \
 	io.methvin:directory-watcher:0.9.10 \
   	org.apache.xmlgraphics:batik-transcoder:1.14 \
-  	org.apache.xmlgraphics:batik-codec:1.14
+  	org.apache.xmlgraphics:batik-codec:1.14 \
```

Comment:
This should be added to the pom.js file, not here, since this file is generated by the pom. See the 'javaDependencies' section of: https://github.com/kgrgreer/foam3/blob/development/pom.js